### ISSUE TITLE
Can't get running on Xcode 7

### ISSUE THREAD

**Issue by thankmelater23** [poboke/ActivatePowerMode#6]
I feel like I'm missing something, but followed the steps to activate except activating it in the plugin menu which doesn't exist in Xcode 7 as far as I know and can see. Can someone tell me what I have to do to get this running on xcode 7?

**Comment by poboke** [comment]
Do you mean that you can not see the `Plugins` menu?
Do you restart the Xcode?

**Comment by thankmelater23** [comment]
I actually ended up finding an alternative that works. And yes I restarted. Thanks

**Comment by vanney9** [comment]
I can't find Plugins on Xcode 7.2.  But when i  was using Xcode 7.1, I can find Plugins.

**Comment by poboke** [comment]
I have added the UUID for Xcode 7.2 Beta.

**Comment by thankmelater23** [comment]
Yes there wasn't a plug in menu available also. In the working alternative I'm using the activation button is in the view menu

**Comment by vanney9** [comment]
I have reinstall it. But can not find the Plugins menu still. And I have updated my Xcode to 7.2 just 2 hours ago, and the Xcode is not 7.2 Beta,  it`s  Xcode 7.2.
<img width="401" alt="2015-12-09 4 59 41" src="https://cloud.githubusercontent.com/assets/8433575/11680942/b93e0c62-9e96-11e5-8cec-52ce8cb830e0.png">

**Comment by vanney9** [comment]
I reinstalled it through Alcatraz, is that the problem?

**Comment by poboke** [comment]
@thankmelater23 
When you restart the Xcode, can you see this window?
![plugin.png](https://cloud.githubusercontent.com/assets/4415086/11682240/81136c4e-9e9e-11e5-8c60-c31e8c15cd3a.png)
You should click the `Load Bundle` button.

**Comment by poboke** [comment]
@vanney9 
I have just added the UUID for Xcode7.2.
Please reinstall it again.

**Comment by vanney9** [comment]
@poboke 
Yeah, it works!!!
